pkg/kudoctl/cmd/update.go

```diff
@@ -57,13 +56,13 @@ func newUpdateCmd() *cobra.Command {
 
 func validateUpdateCmd(args []string, options *updateOptions) error {
 	if len(args) != 0 {
-		return errors.New("expecting no arguments provided for update. Only named flags are accepted")
+		return fmt.Errorf("expecting no arguments provided for update. Only named flags are accepted")
```

Comment:
nit: it's ok to use Go's own `errors.New` here (and other places), since you don't have any arguments besides the message itself. This is what `fmt.Errorf` boils down to (but with more code around it)

Reply:
yeah... I kind of wish go had `errors.Newf` to consolidate error processing instead of scattering it.  Thanks!